src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -72,9 +71,64 @@ describe('ProctoredExamSettings check default on create zendesk ticket field tes
     const zendeskTicketInput = screen.getByTestId('createZendeskTicketsYes');
     expect(zendeskTicketInput.checked).toEqual(true);
   });
+
+  it('Hides all other fields when enabledProctorExam is false when first loaded', async () => {
+    cleanup();
+    auth.getAuthenticatedHttpClient = jest.fn(() => ({
+      get: async () => ({
+        data: {
+          proctored_exam_settings: {
+            enable_proctored_exams: false,
+            allow_proctoring_opt_out: false,
+            proctoring_provider: 'mockproc',
+            proctoring_escalation_email: 'test@example.com',
+            create_zendesk_tickets: true,
+          },
+          available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
+          course_start_date: '2070-01-01T00:00:00Z',
+        },
+      }),
+    }));
+
+    await act(async () => render(<ProctoredExamSettings {...defaultProps} />));
+    await waitFor(() => {
+      screen.getByTestId('enableProctoredExamCheckbox');
+    });
+    const enabledProctoredExamCheck = screen.getByTestId('enableProctoredExamCheckbox');
+    expect(enabledProctoredExamCheck.checked).toEqual(false);
+    expect(screen.queryByText('Allow Opting Out of Proctored Exams')).toBeNull();
```

Comment:
Why not use the `getByLabelText` query here as well?

Reply:
This element does not have a label attribute. It has a text element that's done through `<form.label>` tag, which is not a label attribute when HTML is generated. So, gotta use `queryByText` function here.